projects/miopen/test/gtest/tensor_test.cpp

```
@@ -105,13 +105,16 @@ class CPU_tensor_nxd_NONE : public ::testing::TestWithParam<TestCase>
     static void SetUpTestSuite()
     {
         miopenTensorDescriptor_t tensor;
+        testing::internal::CaptureStderr();
+        auto status        = miopenSet4dTensorDescriptor(nullptr, miopenHalf, 100, 32, 8, 8);
+        std::string output = testing::internal::GetCapturedStderr();
```

We should only capture this for those lines that emit errors (that we expect), and then we should verify that we actually saw the error.  Otherwise, this might hide errors where we did not expect any

---

addressed.